### PR TITLE
Added python email client script

### PR DESCRIPTION
**Pull request by PatelKeviin**
Issue: [#94 ](https://github.com/ankitdobhal/Awesome-Python-Scripts/issues/94)

About PR:
Need to send emails daily with the same email message to the same pool of senders? If yes, then this script will help to automate the process of sending emails automatically.

### CODE REVIEW

File: System-Automation-Scripts/Email_Client/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Email Client using Python for automating email shooting process
```
Comment (Neilblaze) [inline System-Automation-Scripts/Email_Client/README.md:1]: I will recommend you to elaborate the installation process briefly along with what's already present in README file.

Rest, LGTM :+1:

**Review by Neilblaze — CHANGES_REQUESTED** [review]

**Comment by Neilblaze** [comment]
Also, nice & clean code @PatelKeviin !
Great job :100:

File: System-Automation-Scripts/Email_Client/README.md (hunk 1 above)
Comment (PatelKeviin) [reply System-Automation-Scripts/Email_Client/README.md:1]: Hi @Neilblaze, I have updated the changes as advised. Also, thank you very much for your kind words!

Comment (PatelKeviin) [reply System-Automation-Scripts/Email_Client/README.md:1]: @Neilblaze as all the required modules are part of Python Standard Libraries, nothing in particular is required in terms of set-up as long as the user has Python installed on his/her machine :)

**Pull request merged**